backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  phoneNumberType            String?       @map("phone_number_type")
+  noPhone                    Boolean?      @map("no_phone")
+  sameAddress                YesNoEnum     @map("same_address")
+  relationship               String?
+  workInRegion               YesNoEnum     @map("work_in_region")
+  addressId                  String?       @unique() @map("address_id") @db.Uuid
+  workAddressId              String?       @unique() @map("work_address_id") @db.Uuid
+  applicationId              String?       @map("application_id") @db.Uuid
+  applications               Applications? @relation(fields: [applicationId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  householdMemberAddress     Address?      @relation("household_member_address", fields: [addressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  householdMemberWorkAddress Address?      @relation("household_member_work_address", fields: [workAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@index([applicationId])
+  @@map("household_member")
+}
+
+// Note: [name] formerly max length 256
+model Jurisdictions {
+  id                          String                   @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt                   DateTime                 @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                   DateTime                 @updatedAt @map("updated_at") @db.Timestamp(6)
+  name                        String                   @unique()
+  notificationsSignUpUrl      String?                  @map("notifications_sign_up_url")
+  languages                   LanguagesEnum[]          @default([en])
+  partnerTerms                String?                  @map("partner_terms")
+  publicUrl                   String                   @default("") @map("public_url")
+  emailFromAddress            String?                  @map("email_from_address")
+  rentalAssistanceDefault     String                   @map("rental_assistance_default")
+  enablePartnerSettings       Boolean                  @default(false) @map("enable_partner_settings")
+  enableAccessibilityFeatures Boolean                  @default(false) @map("enable_accessibility_features")
+  enableUtilitiesIncluded     Boolean                  @default(false) @map("enable_utilities_included")
+  amiChart                    AmiChart[]
+  multiselectQuestions        MultiselectQuestions[]
+  listings                    listings[]
+  reservedCommunityTypes      ReservedCommunityTypes[]
+  translations                Translations[]
+  user_accounts               UserAccounts[]
+
+  @@map("jurisdictions")
+}
+
+model ListingEvents {
+  id        String            @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt DateTime          @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt DateTime          @updatedAt @map("updated_at") @db.Timestamp(6)
+  type      MarketingTypeEnum
```

Comment:
I don't think this is the right enum "ListingEventType"

Reply:
Sheeesh
nice catch